tests/test_envs.py

```diff
@@ -26,7 +26,11 @@
 ]
 
 ATARI_ENVS: List[str] = [
-    _seals_name(gym_spec) for gym_spec in seals.GYM_ATARI_ENV_SPECS
+    _seals_name(gym_spec, masked=False) for gym_spec in seals.GYM_ATARI_ENV_SPECS
```

Comment:
Maybe define the masked/unmasked envs as separate constants? and then combine them. Probably more readable and it would also prevent black from breaking the lines in strange ways.

Reply:
Good idea! Done.